tests/Fixtures/Config/database.php

```diff
@@ -0,0 +1,12 @@
+<?php
```

Comment:
This file should be in gitignore, because our testsuite will generate it based on whether we're testing sqlite, mysql or postgres. Is there a reason you added it in VCS?

Reply:
No clue, I think this is a rebasing issue because I rebased onto the newer version which uses database.config.php. Will remove it.